tests/test_requests.py

```diff
@@ -518,7 +518,8 @@ def test_basicauth_encodes_byte_strings(self):
         ))
     def test_errors(self, url, exception):
         with pytest.raises(exception):
-            requests.get(url, timeout=1)
+            r = requests.get(url, timeout=1)
+            assert r.exception == exception
```

Comment:
Sorry if i'm not following normal convention.  Is there a method you'd like me to use to test this?